### ISSUE TITLE
SRT input sources don't recover in fallback

### ISSUE THREAD

**Issue by Shizmob** [savonet/liquidsoap#1186]
**Describe the bug**
When using an SRT input source together with `mksafe()`, it seems to play just fine the first time I connect to it, but if I disconnect and reconnect, it seems to never recover from the fallback and start playing again, leaving just silence.

**To Reproduce**
Liquidsoap script:
```
srt = (input.srt(port=9000, clock_safe=false):source(2,1,0))
out(mksafe(srt))
```

ffmpeg commandline:
```
ffmpeg -re -i <my-file.mp4> -f mpegts "srt://localhost:9000?pkt_size=1316"
```

**Expected behavior**
When I connect again after the first disconnect, for it to start playing the audio I stream to it again.

**Version details**
 - OS: macOS 10.15.1
 - Version: 1.5.0+git@34c16a76

**Install method**
From source, by opam package pinning.

**Debug log**
https://txt.shiz.me/MmMyZDM1ND.txt

**Comment by toots** [comment]
Thanks for this report. We should get back to it when the current work on video support has stabilized in the main branch.

**Comment by toots** [comment]
This issue was fixed in aa38e411dd84c783b09c7b4fa8393f7593c8420d for `v2.0.0` and 83dda4672219f7b98354cea3effa8920af16daa5 for `main`. Sorry it took so long, tanks for reporting!
